template.yaml

```diff
@@ -16,11 +16,16 @@ spec:
         - component
         - owner
         - slack_contact_channel
+        - jenkins_topic
       properties:
         product:
           title: Product
           type: string
           description: Product this component belongs to, normally the team name, e.g. cmc
+        jenkins_topic:
+          title: Jenkins Topic
+          type: string
+          description: GitHub Topic added in order for Jenkins to find the repository when it scans GitHub, alphabetically grouped e.g. jenkins-cft-j-z
```

Comment:
is there a way to set a default value? should have the other available values here too